src/UniOneClient.php

```diff
@@ -11,110 +11,134 @@
-   *
-   * @return string
-   * @throws \GuzzleHttp\Exception\GuzzleException
-   */
-  public function sender(array $body, array $head = []): string {
-    $responseHeader = [
-      'Content-Type' => 'application/json',
-      'Accept' => 'application/json',
-      'X-API-KEY' => $this->apiKey,
-      'X-Mailer' => 'UniOne SDK-PHP agent'
-    ];
+    /**
+     * Set the timeout in seconds.
+     *
+     * @param float $timeout the timeout in seconds
+     */
+    public function setTimeout(float $timeout): void
+    {
+        $this->timeout = $timeout;
+    }
 
-    // Build body for request.
-    // @todo: Refactor this and create a special class for it.
-    $responseBody = [
-      "message" => [
-        "recipients" => [
-          [
-            "email" => $body['recipients_email'] ?? NULL,
-            "substitutions" => $body['recipients_substitutions'] ?? NULL,
-            "metadata" => $body['recipients_metadata'] ?? NULL,
-          ]
-        ],
-        "template_id" => $body['template_id'] ?? NULL,
-        "skip_unsubscribe" => $body['skip_unsubscribe'] ?? NULL,
-        "global_language" => $body['global_language'] ?? NULL,
-        "template_engine" => $body['template_engine'] ?? NULL,
-        "global_substitutions" => $body['global_substitutions'] ?? NULL,
-        "global_metadata" => $body['global_metadata'] ?? NULL,
-        "body" => [
-          "html" => $body['body_html'] ?? NULL,
-          "plaintext" => $body['body_plaintext'] ?? NULL,
-          "amp" => $body['body_amp'] ?? NULL,
-        ],
-        "subject" => $body['subject'] ?? NULL,
-        "from_email" => $body['from_email'] ?? NULL,
-        "from_name" => $body['from_name'] ?? NULL,
-        "reply_to" => $body['reply_to'] ?? NULL,
-        "track_links" => 0,
-        "track_read" => 0,
-        "headers" => $body['headers'] ?? NULL,
-        "attachments" => $body['attachments'] ?? NULL,
-        "inline_attachments" => $body['inline_attachments'] ?? NULL,
-        "options" => $body['options'] ?? NULL,
-      ]
-    ];
+    /**
+     * Send a request to the UniOne API.
+     *
+     * @param array $body the request body
+     * @param array $head the request headers
+     *
+     * @return string
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function sender(array $body, array $head = []): string
+    {
+        $responseHeader = [
+          'Content-Type' => 'application/json',
+          'Accept' => 'application/json',
+          'X-API-KEY' => $this->apiKey,
+          'X-Mailer' => 'UniOne SDK-PHP agent',
+        ];
 
-    try {
-      // Send request.
-      $response = $this->httpClient->post('email/send.json', [
-          'headers' => $head + $responseHeader,
-          'json' => $responseBody
-        ]
-      );
+        // Build body for request.
+        // @todo: Refactor this and create a special class for it.
+        $responseBody = [
+          'message' => [
+            'recipients' => [
```

Comment:
It seems we do not support multiple recipients because each of them should be its own array. 
Should we support them for the first release?